### PR TITLE
SAK-46998 Move datepicker webcomponent to datetime-local

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-46998

### CODE REVIEW

**Comment by ottenhoff** [comment]
Nice! For anyone else curious: https://caniuse.com/?search=datetime-local

File: webcomponents/tool/src/main/frontend/package.json [hunk 1]
```diff
@@ -24,7 +24,6 @@
     "@popperjs/core": "2.9.3",
     "@webcomponents/webcomponentsjs": "2.6.0",
     "cropperjs": "1.5.12",
-    "flatpickr": "4.6.9",
```
Comment (mpellicer) [inline webcomponents/tool/src/main/frontend/package.json:27]: Do you know if the flatpickr webjar can be removed from the library project?

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/package.json:27]: that's a good question. This was just using it from node_modules, so is unrelated.

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/package.json:27]: Actually, I see what you mean. Assignments brings it in. Maybe that was a leftover from before I moved to the npm module.

**Review by ern — APPROVED** [review]

**Pull request merged**